tikv-configuration-file.md

```diff
@@ -369,8 +369,12 @@ Configuration items related to storage.
 
 ### `enable-ttl`
 
+> **Warning:**
+>
+> - You can set the value of `enable-ttl` to `true` or `false` **only when** deploying a new TiKV cluster. **DO NOT** modify the value of this configuration item for an existing TiKV cluster. The data stored in TiKV clusters configured this item to `true` and `false` has different formats. Therefore, if you modify the value of this item in an existing TiKV cluster, the cluster will store data in different formats, causing TiKV to report the "can't enable ttl on a non-ttl" error when you restart the TiKV cluster.
+> - You can use `enable-ttl` in a TiKV cluster only. **DO NOT** use this configuration item in a cluster with TiDB nodes, which means setting `enable-ttl` to `true` in such clusters. Otherwise, serious consequences might occur, such as data corruption and the upgrade failure of TiDB clusters.
```

Comment:
```suggestion
> - Use `enable-ttl` **ONLY IN** a TiKV cluster. **DO NOT** use this configuration item in a cluster that has TiDB nodes (which means setting `enable-ttl` to `true` in such clusters). Otherwise, critical issues such as data corruption and the upgrade failure of TiDB clusters will occur.
```